src/oneD/StFlow.cpp

```diff
@@ -199,6 +195,7 @@ void StFlow::resize(size_t ncomponents, size_t points)
     }
     m_flux.resize(m_nsp,m_points);
     m_wdot.resize(m_nsp,m_points, 0.0);
+    m_hk.resize(m_nsp,m_points, 0.0);
```

Comment:
```suggestion
    m_hk.resize(m_nsp ,m_points, 0.0);
```
While you used formatting that matches pre-existing code, the style guide should be followed for new additions (here and elsewhere).

Reply:
Corrected.